### DISCUSSION TITLE
Initial primal_bound and dual_bound

### DISCUSSION THREAD

**Discussion by ferraric** [ds4dm/ml4co-competition#98] [Q&A]
This question is aimed only at the configuration task (even though the issue might be applicable to the other tasks).

In the dataset, we are given access to an initial primal_bound and dual_bound for every problem instance. That information is not passed on to the agent that we are allowed to control (meaning the ObservationFunction and Policy classes). Am I correct in assuming that we are not meant to use this information for choosing an action (aka solver parameters)?

**Comment by gasse** [comment]
Hi @ferraric ,

Yes that's right, this information (initial primal bound, initial dual bound) is meant only for the computation of the evaluation metric, and is not supposed to be used by your agent for choosing an action.

Best,
Maxime
